modules/my_health/spec/requests/my_health/v1/medical_records/allergies_spec.rb

```diff
@@ -14,6 +14,15 @@
   let(:va_patient) { true }
   let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
 
+  VCR.configure do |config|
+    config.register_request_matcher :wildcard_path do |request1, request2|
+      # Remove the user id and icn after `/isValidSMUser/` to handle any user id and icn
+      path1 = request1.uri.gsub(%r{/isValidSMUser/.*}, '/isValidSMUser')
+      path2 = request2.uri.gsub(%r{/isValidSMUser/.*}, '/isValidSMUser')
+      path1 == path2
+    end
+  end
```

Comment:
Try moving this into the ClientHelpers mixin

Reply:
done